pkg/fileutils/union/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Why did we copy the package to go-base?

Reply:
because it isn't actually posted as a module anywhere, the last time i checked